pkg/controller/elasticsearch/services/services.go

```diff
@@ -50,6 +50,7 @@ func NewTransportService(es esv1.Elasticsearch) *corev1.Service {
 	labels := label.NewLabels(nsn)
 	ports := []corev1.ServicePort{
 		{
+			Name:     "tls-transport",
```

Comment:
A note for any other reviewers: istio [requires](https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/#manual-protocol-selection) it be prefixed with the protocol